docs/release-notes/NuGet-6.14.md

```diff
@@ -2,27 +2,130 @@
 title: NuGet 6.14 Release Notes
 description: Release notes for NuGet 6.14 including new features, bug fixes, and DCRs.
 author: zivkan
+ms.date: 5/6/2025
 ms.topic: conceptual
 ---
-# NuGet 6.14 Release Notes
 
-<!--
-    This is intentionally not yet added to TOC.md.
-    When this version is ready for release:
-    1. Rename this file to the next version
-    2. Change the version strings in this file to the next version
-    3. Use the release tool to create the real release notes for the version ready for release
-    4. Add the release notes to TOC.yml
--->
+# NuGet 6.14 Release Notes
 
 NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**6.14**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.14](https://visualstudio.microsoft.com/downloads/) | [9.0.2xx](https://dotnet.microsoft.com/download/dotnet/9.0)<sup>1</sup> |
+| [**6.14.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.14.1](https://visualstudio.microsoft.com/downloads/) | [9.0.300](https://dotnet.microsoft.com/download/dotnet/9.0)<sup>1</sup> |
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.14.0
+
+* Event tracing in new dependency resolver - [#14134](https://github.com/NuGet/Home/issues/14134)
+
+* Support for new slnx format in static graph-based restore - [#14086](https://github.com/NuGet/Home/issues/14086)
+
+* Add Net90 to FrameworkConstants.CommonFrameworks - [#14067](https://github.com/NuGet/Home/issues/14067)
+
+* NuGet needs to onboard to Unified Settings and create General page - [#14040](https://github.com/NuGet/Home/issues/14040)
+
+* Support for new `slnx` solution format - [#14034](https://github.com/NuGet/Home/issues/14034)
+
+* dotnet-nuget-push: does not support --config-file included in examples - [#4879](https://github.com/NuGet/Home/issues/4879)
```

Comment:
Maybe we can rename this to something like dotnet nuget push now supports --config-file?